core/src/main/java/org/arquillian/smart/testing/spi/TestExecutionPlannerFactory.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.smart.testing.spi;
+
+import java.io.File;
+
+public interface TestExecutionPlannerFactory {
+
+    String alias();
+
+    boolean isFor(String name);
+
+    TestExecutionPlanner create(File projectDir, String[] globPatterns);
```

Comment:
It's not optimal that we pass `globPatterns` around, but there is potentially bigger improvement needed in this area, so let's do refactor it as a part of #8